odbc/src/handles.rs

```diff
@@ -1,12 +1,20 @@
-use std::{collections::HashSet, sync::RwLock};
+use crate::util::*;
+use odbc_sys::{Integer, SmallInt, SqlReturn, WChar};
+use std::{
+    collections::HashSet,
+    fmt::{Display, Formatter},
+    sync::RwLock,
+};
 
-use odbc_sys::Integer;
+pub const VENDOR_IDENTIFIER: &str = "MongoDB";
```

Comment:
A few questions:
- It looks like this is currently only used in one place. Do we plan to use it elsewhere in the future? If not, it might make sense to just inline it there
- Is there any need for this to be `pub`?

Reply:
Yeah, we'll need to reuse this in the future because it'll be used in every error message instance. I'll make it private!